README.md

```diff
@@ -12,30 +12,59 @@ npm install eslint-config-brightspace
 
 ## Usage
 
-Set the `extends` property in the `.eslintrc.json` file, replacing `<environment>` with the desired environment-specific config.
+Shared configuration, they need to exported from the `eslint.config.js` file on the root directory.
+```js
+export { nodeConfig as default } from 'eslint-config-brightspace/index.js';
```

Comment:
`index.js` shouldn't be necessary since it's the main entrypoint for the package.
```suggestion
export { nodeConfig as default } from 'eslint-config-brightspace';
```